libs/vkd3d/command.c

```diff
@@ -11673,6 +11674,7 @@ static void *d3d12_command_queue_submission_worker_main(void *userdata)
     VKD3D_REGION_DECL(queue_signal);
     VKD3D_REGION_DECL(queue_execute);
 
+    VKD3D_PROFILE_THREAD_NAME("vkd3d_queue");
```

Comment:
Couldn't this be part of vkd3d_set_thread_name()?

Reply:
Yes it could. My thinking was to pass the static constant directly in to the function for performance but that is unnecessary for thread names.